config/config.example.yml

```diff
@@ -502,7 +502,22 @@ notifyMetrics:
     config: 'NOTIFY.outgoing.delivered'
     description: 'admin-notify-dashboard.NOTIFY.outgoing.delivered.description'
 
-
+# Creative Commons metadata fields
+ccLicense:
+  # Icon variant:
+  # 'full' variant shows image, a disclaimer (optional) and name (always), better for the item page content.
+  # 'small' (default) variant shows image and name (optional), better for the item page sidebar
+  variant: small
+  # Field name containing the CC license URI, as configured in the back-end, in the 'dspace.cfg' file, property 'cc.license.uri'
+  # Defaults to 'dc.rights.uri'
+  uriField: dc.rights.uri
+  # Field name containing the CC license name, as configured in the back-end, in the 'dspace.cfg' file, property 'cc.license'
+  # Defaults to 'dc.rights'
+  nameField: dc.rights
```

Comment:
Similar to the note above, we also could remove this config and instead configure the REST API to expose `cc.license.name` and then call ``
```
this.configService.findByPropertyName('cc.license.name')
```
Again, your approach here will work too...but it would require users to make sure these two settings are duplicated into both dspace.cfg and config.yml.